lib/IDE/ModuleInterfacePrinting.cpp

```diff
@@ -686,14 +696,15 @@ void swift::ide::printModuleInterface(
 
     for (auto CM : ClangModules) {
       for (auto DeclAndLoc : ClangDecls[CM.first])
-        PrintDecl(DeclAndLoc.first);
+        if (PrintDecl(DeclAndLoc.first) && Options.EmptyLineBetweenDecls)
+          Printer.printNewline();
```

Comment:
This is the main change of this PR. Newline after printing a decl just like Swift decls few lines below